recipes/recipes_emscripten/pixman/build.sh

```diff
@@ -9,4 +9,4 @@ meson setup builddir \
     --cross-file=$RECIPE_DIR/emscripten.meson.cross
 
 ninja -v -C builddir -j ${CPU_COUNT}
-ninja -C builddir install -j ${CPU_COUNT}
\ No newline at end of file
+ninja -C builddir install -j ${CPU_COUNT}
```

Comment:
Pixman's meson build needs to be patched too.

https://gitlab.freedesktop.org/pixman/pixman/-/blob/master/meson.build?ref_type=heads#L457

Reply:
Done